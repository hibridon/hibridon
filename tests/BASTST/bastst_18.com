inp=bastst_18.inp
out=bastst_18.out
run
save=save_18.inp