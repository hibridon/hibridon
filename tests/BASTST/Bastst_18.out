      BASTST                                     
 DUMMY POTENTIAL FOR TESTING PURPOSES            
 ==============================
     *** Parameters:
 JTOT1  =  20        JTOT2  =  20        JTOTD  =   5        JLPAR  =   1
 NERG   =   1        NUMAX  =   0        NUMIN  =   0        NUD    =   1
 LSCREEN=  48        IPRINT =   1
 FSTFAC =  15.00     RINCR  =  1.000     RCUT   =  30.00     RENDAI =  1.000    
 RENDLD =  1.000     RSTART =  1.000     SPAC   = 0.1000     TOLAI  =  1.000    
 XMU    =  16.47    
     *** SYM-TOP- system parameters:
 NTERM  =   4        NUMPOT =   0        IPOTSY =   3        IOP    =  -1
 JMAX   =   5
 BROT   =  9.578     CROT   =  4.742     EMAX   =  800.0    
 LAMMIN:    0   3   6   9
 LAMMAX:    8   9   8   9
 MPROJ:     0   3   6   9
     *** Flags:
 AIRYFL= F    BASTST= T    BATCH = F    CHLIST= T    CSFLAG= F    FLAGHF= F
 FLAGSU= F    IHOMO = F    IPOS  = F    LOGDFL= F    NOPRIN= F    NUCROS= F
 PHOTOF= F    PRAIRY= F    PRLOGD= F    PRPART= F    PRSMAT= F    PRT2  = F
 PRXSEC= F    READPT= F    RSFLAG= F    T2TEST= F    TWOMOL= F    WAVEFL= F
 WRPART= F    WRSMAT= F    WRXSEC= F    BOUNDC= F
 ** Energies:     5000.000000
 NOUT: 13; JOUT:  0   1   2   3   4   5   6   7   8   9  10  11  12
 INDOUT:    0   1  -1   2  -2   3  -3   4  -4
 ==============================

 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V 5.00

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
CMAKE BUILD DATE: 2023-03-19 at 06:11
CMAKE BUILD TYPE: Debug
CODE GIT REVISION: 97c80025011c4db98875e99113c6dbea79108cf7
SYS: Darwin 22.2.0 x86_64
COMPILER: /opt/intel/oneapi/compiler/2022.2.1/mac/bin/intel64/ifort
COMPILE OPTIONS: 
 --------------------------------------------------------------------------
            CURRENT HARDWARE CONFIGURATION:
 
      Model Name: MacBook Pro
      Model Identifier: MacBookPro14,3
      Processor Name: Quad-Core Intel Core i7
      Processor Speed: 2.9 GHz
      Number of Processors: 1
      Total Number of Cores: 4
      L2 Cache (per Core): 256 KB
      L3 Cache: 8 MB
      Hyper-Threading Technology: Enabled
      Memory: 16 GB
 --------------------------------------------------------------------------

 -----------------------------------------------------------------------------
 All publications resulting from use of the integrators included in the
 Hibridon code must include the following reference:

 D. E. Manolopoulos, J. Chem. Phys. 85, 6425 (1986); M. H. Alexander and D. E.
 Manolopoulos, J. Chem. Phys. 80, 2044 (1987).

 All publications involving the determination ofphotodissociation cross sections
 must also include the following reference:

 M. H. Alexander, Comput. Phys. Commun, 75, 87 (1993).

 All publications investigating flux redistribution must include the following references:

 M. H. Alexander, J. Chem. Phys. 95, 8931 (1991); D. E. Manolopoulos and
 M. H. Alexander, J. Chem. Phys.  97, 2527 (1992).

 All publications resulting from use of the Hibridon package must include
 the following reference:

 HIBRIDON is a package of programs for the time-independent quantum treatment
 of inelastic collisions and photodissociation written by M. H. Alexander,
 D. E.  Manolopoulos, H.-J. Werner, and B. Follmeg, with contributions by
 P. F. Vohralik, D. Lemoine, G. Corey, B. Johnson, T. Orlikowski, A. Berning,
 A. Degli-Esposti, C. Rist, P. Dagdigian, B. Pouilly, G. van der Sanden, M. Yang, F. de Weerd, S. Gregurick, J. Klos, and F. Lique
 -----------------------------------------------------------------------------

 ** LABEL:          BASTST                                     
 ** POT NAME:  DUMMY POTENTIAL FOR TESTING PURPOSES            
 ** DATE:      Mar 19 06:12:17 2023

 *** WARNING:  NAIRY .NE. 1 BUT AIRYFL .EQ. .FALSE.
 ** NMAX= 500  NAIRY= 500
 ** TOTAL NUMBER OF ANISTROPIC TERMS IN POTENTIAL = 20

 **  CC SYMMETRIC TOP **
     RMU=  16.4700  A=BROT=  9.578  CROT=  4.742
     E=5000.00  JTOT=  20   JLPAR= 1

 OPEN CHANNELS ELIMINATED WHICH ARE CLOSED AT R=   30.00

   N   J  EPS  K   L    EINT(CM-1)
   1   0   1   0  20     0.000
   2   1   1   1  20    14.320
   3   1  -1   1  19    14.320
   4   1  -1   1  21    14.320
   5   1   1   0  19    19.156
   6   1   1   0  21    19.156
   7   2   1   2  18    38.124
   8   2   1   2  20    38.124
   9   2   1   2  22    38.124
  10   2  -1   2  19    38.124
  11   2  -1   2  21    38.124
  12   2   1   1  19    52.631
  13   2   1   1  21    52.631
  14   2  -1   1  18    52.631
  15   2  -1   1  20    52.631
  16   2  -1   1  22    52.631
  17   2   1   0  18    57.467
  18   2   1   0  20    57.467
  19   2   1   0  22    57.467
  20   3   1   3  18    71.412
  21   3   1   3  20    71.412
  22   3   1   3  22    71.412
  23   3  -1   3  17    71.412
  24   3  -1   3  19    71.412
  25   3  -1   3  21    71.412
  26   3  -1   3  23    71.412
  27   3   1   2  17    95.591
  28   3   1   2  19    95.591
  29   3   1   2  21    95.591
  30   3   1   2  23    95.591
  31   3  -1   2  18    95.591
  32   3  -1   2  20    95.591
  33   3  -1   2  22    95.591
  34   3   1   1  18   110.099
  35   3   1   1  20   110.099
  36   3   1   1  22   110.099
  37   3  -1   1  17   110.099
  38   3  -1   1  19   110.099
  39   3  -1   1  21   110.099
  40   3  -1   1  23   110.099
  41   4   1   4  16   114.184
  42   4   1   4  18   114.184
  43   4   1   4  20   114.184
  44   4   1   4  22   114.184
  45   4   1   4  24   114.184
  46   4  -1   4  17   114.184
  47   4  -1   4  19   114.184
  48   4  -1   4  21   114.184
  49   4  -1   4  23   114.184
  50   3   1   0  17   114.935
  51   3   1   0  19   114.935
  52   3   1   0  21   114.935
  53   3   1   0  23   114.935
  54   4   1   3  17   148.035
  55   4   1   3  19   148.035
  56   4   1   3  21   148.035
  57   4   1   3  23   148.035
  58   4  -1   3  16   148.035
  59   4  -1   3  18   148.035
  60   4  -1   3  20   148.035
  61   4  -1   3  22   148.035
  62   4  -1   3  24   148.035
  63   5   1   5  16   166.440
  64   5   1   5  18   166.440
  65   5   1   5  20   166.440
  66   5   1   5  22   166.440
  67   5   1   5  24   166.440
  68   5  -1   5  15   166.440
  69   5  -1   5  17   166.440
  70   5  -1   5  19   166.440
  71   5  -1   5  21   166.440
  72   5  -1   5  23   166.440
  73   5  -1   5  25   166.440
  74   4   1   2  16   172.214
  75   4   1   2  18   172.214
  76   4   1   2  20   172.214
  77   4   1   2  22   172.214
  78   4   1   2  24   172.214
  79   4  -1   2  17   172.214
  80   4  -1   2  19   172.214
  81   4  -1   2  21   172.214
  82   4  -1   2  23   172.214
  83   4   1   1  17   186.722
  84   4   1   1  19   186.722
  85   4   1   1  21   186.722
  86   4   1   1  23   186.722
  87   4  -1   1  16   186.722
  88   4  -1   1  18   186.722
  89   4  -1   1  20   186.722
  90   4  -1   1  22   186.722
  91   4  -1   1  24   186.722
  92   4   1   0  16   191.558
  93   4   1   0  18   191.558
  94   4   1   0  20   191.558
  95   4   1   0  22   191.558
  96   4   1   0  24   191.558
  97   5   1   4  15   209.963
  98   5   1   4  17   209.963
  99   5   1   4  19   209.963
 100   5   1   4  21   209.963
 101   5   1   4  23   209.963
 102   5   1   4  25   209.963
 103   5  -1   4  16   209.963
 104   5  -1   4  18   209.963
 105   5  -1   4  20   209.963
 106   5  -1   4  22   209.963
 107   5  -1   4  24   209.963
 108   5   1   3  16   243.814
 109   5   1   3  18   243.814
 110   5   1   3  20   243.814
 111   5   1   3  22   243.814
 112   5   1   3  24   243.814
 113   5  -1   3  15   243.814
 114   5  -1   3  17   243.814
 115   5  -1   3  19   243.814
 116   5  -1   3  21   243.814
 117   5  -1   3  23   243.814
 118   5  -1   3  25   243.814
 119   5   1   2  15   267.993
 120   5   1   2  17   267.993
 121   5   1   2  19   267.993
 122   5   1   2  21   267.993
 123   5   1   2  23   267.993
 124   5   1   2  25   267.993
 125   5  -1   2  16   267.993
 126   5  -1   2  18   267.993
 127   5  -1   2  20   267.993
 128   5  -1   2  22   267.993
 129   5  -1   2  24   267.993
 130   5   1   1  16   282.501
 131   5   1   1  18   282.501
 132   5   1   1  20   282.501
 133   5   1   1  22   282.501
 134   5   1   1  24   282.501
 135   5  -1   1  15   282.501
 136   5  -1   1  17   282.501
 137   5  -1   1  19   282.501
 138   5  -1   1  21   282.501
 139   5  -1   1  23   282.501
 140   5  -1   1  25   282.501
 141   5   1   0  15   287.337
 142   5   1   0  17   287.337
 143   5   1   0  19   287.337
 144   5   1   0  21   287.337
 145   5   1   0  23   287.337
 146   5   1   0  25   287.337
ILAM=  1 LAM=  0 MU=  0 LAMNUM(ILAM) =    146
ILAM=  2 LAM=  1 MU=  0 LAMNUM(ILAM) =    280
ILAM=  3 LAM=  2 MU=  0 LAMNUM(ILAM) =    565
ILAM=  4 LAM=  3 MU=  0 LAMNUM(ILAM) =    672
ILAM=  5 LAM=  4 MU=  0 LAMNUM(ILAM) =    861
ILAM=  6 LAM=  5 MU=  0 LAMNUM(ILAM) =    836
ILAM=  7 LAM=  6 MU=  0 LAMNUM(ILAM) =    859
ILAM=  8 LAM=  7 MU=  0 LAMNUM(ILAM) =    744
ILAM=  9 LAM=  8 MU=  0 LAMNUM(ILAM) =    647
ILAM= 10 LAM=  3 MU=  3 LAMNUM(ILAM) =    917
ILAM= 11 LAM=  4 MU=  3 LAMNUM(ILAM) =   1100
ILAM= 12 LAM=  5 MU=  3 LAMNUM(ILAM) =   1074
ILAM= 13 LAM=  6 MU=  3 LAMNUM(ILAM) =   1111
ILAM= 14 LAM=  7 MU=  3 LAMNUM(ILAM) =   1048
ILAM= 15 LAM=  8 MU=  3 LAMNUM(ILAM) =    816
ILAM= 16 LAM=  9 MU=  3 LAMNUM(ILAM) =    520
ILAM= 17 LAM=  6 MU=  6 LAMNUM(ILAM) =    664
ILAM= 18 LAM=  7 MU=  6 LAMNUM(ILAM) =    578
ILAM= 19 LAM=  8 MU=  6 LAMNUM(ILAM) =    532
ILAM= 20 LAM=  9 MU=  9 LAMNUM(ILAM) =    110
 ** TOTAL NUMBER OF NONZERO V2 MATRIX ELEMENTS IS 14080
 ** BASTST=.TRUE.; TEST OF SUBROUTINE BASIS
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:     0
      TIMING:  ELAPSED 00:00:00 / CPU 00:00:00 / MP RATIO    6.1 %
      CURRENT DATE:  Mar 19 06:12:17 2023
 ===============================================================================
